wp-parsely.php

```diff
@@ -17,6 +17,8 @@
 
 namespace Automattic\VIP\WP_Parsely_Integration;
 
+use Parsely\Parsely;
+
```

Comment:
This can be removed with my other suggested change.

```suggestion
```

Reply:
Addressed in https://github.com/Automattic/vip-go-mu-plugins/pull/5051/commits/053a7d6895695604ad842bf72e735ebdb13917a1